### ISSUE TITLE
turbinia processor test failing on Fedora Core 30 virtualenv

### ISSUE THREAD

**Issue by joachimmetz** [log2timeline/dftimewolf#165]
```
======================================================================
ERROR: lib.processors.turbinia (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: lib.processors.turbinia
Traceback (most recent call last):
  File "/usr/lib64/python3.7/unittest/loader.py", line 436, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib64/python3.7/unittest/loader.py", line 377, in _get_module_from_name
    __import__(name)
  File "/home/user/Projects/dftimewolf/dftimewolf/tests/lib/processors/turbinia.py", line 19, in <module>
    from dftimewolf.lib.processors import turbinia
  File "./dftimewolf/lib/processors/turbinia.py", line 11, in <module>
    from turbinia import client as turbinia_client
  File "/home/user/Projects/dftimewolf/lib/python3.7/site-packages/turbinia/client.py", line 29, in <module>
    from turbinia import task_manager
  File "/home/user/Projects/dftimewolf/lib/python3.7/site-packages/turbinia/task_manager.py", line 30, in <module>
    import psq
  File "/home/user/Projects/dftimewolf/lib/python3.7/site-packages/psq/__init__.py", line 24, in <module>
    from .broadcast_queue import BroadcastQueue
  File "/home/user/Projects/dftimewolf/lib/python3.7/site-packages/psq/broadcast_queue.py", line 20, in <module>
    from . import queue
  File "/home/user/Projects/dftimewolf/lib/python3.7/site-packages/psq/queue.py", line 37
    name='default', storage=None, extra_context=None, async=True):
                                                          ^
SyntaxError: invalid syntax
```

**Comment by joachimmetz** [comment]
```
pip list
...
psq                      0.7.0
```

**Comment by tomchop** [comment]
This is probably because that version of psq is not python3 compatible. Should be solved with #152

**Comment by joachimmetz** [comment]
has been solved for me, closing issue
